app/commands/solution/generate_readme_file.rb

```diff
@@ -0,0 +1,81 @@
+class Solution
+  class GenerateReadmeFile
+    include Mandate
+
+    initialize_with :solution
+
+    def call
+      [preamble, introduction, instructions, source].compact_blank.join("\n\n")
+    end
+
+    private
+    def preamble
+      if solution.git_exercise.hints.present?
+        hints_text = "If you get stuck on the exercise, check out `HINTS.md`, but try and solve it without using those first :)" # rubocop:disable Layout/LineLength
```

Comment:
Can you move this into i18n pls? Probably a new file for it (see notifications for an example of how to do this - look at `config/locales` and great for `I18n` in `app/models`

Reply:
Only the hints text, or also the other strings in this file?

Reply:
All the strings in this file.

Reply:
Mainly from a seperating data/code POV, rather than me thinking about i18n specifically at this point.

Reply:
I've added some basic localization. Let me know what you think